packages/table/src/regions.ts

```diff
@@ -548,6 +544,42 @@ export class Regions {
         }
     }
 
+    private static maybeFullyOverlapsRegion(regions: IRegion[],
```

Comment:
@giladgray Yeah, I wasn't sure what to call this. Given that `intervalCompareFn` was parameterized, it's sometimes checking for full overlap and sometimes checking for partial overlap. LMK if you have suggestions. Maybe just `findRegion`?